meetings/2017-13-04.md

```diff
@@ -0,0 +1,98 @@
+May 4-5 in Berlin, hope to see you there
+Awesome way to get everyone in the same room
+Roll call:  Myles, William, Bryan, Ashley, Rachel
+
+Agenda to submit:
+
+- Can we get everyone together to talk Moderation? Scope should cover more space. How CommComm can help.
+- What communities can we help with and how do we connect?
+- Meta-contribution session–let’s find a way to bridge the gap between Code + Learn transitioning the afternoon into Collab Summit in the Fall
+- Moderation training
+
+Let’s brainstorm more ideas in the issue, too.
+
+### governance: who has voting rights? [#21](https://github.com/nodejs/community-committee/issues/21)
+For the time being, can we put this on hold? The members have voting rights. What are these external groups and how they want to weigh in on things? 
+
+Another question: do we need to have the distinction between the two? 
+
+_To do_: If we leave the charter as-is(and we just let members vote), would we be violating the charter? (@williamkapke)
+
+
+### Add some scope (aka: stake your claim!) [#13](https://github.com/nodejs/community-committee/issues/13)
+Rachel White would love to see a number of orgs participate or be in the community committee:
+https://github.com/nodejs/code-and-learn
+https://github.com/nodejs/education
+https://github.com/nodejs/evangelism
+https://github.com/nodejs/inclusivity
+
+Our hope is that TSC and CTC feel a part of the Community Committee. This is important to stress and that repos being a part of this is not a landgrab. Outreach and talking about interest in participating is important. We aren’t inviting people to move over without them explicitly stating as a group that they’d like to. 
+
+To do: outreach to groups listed in this issue to gauge interest (@hackygolucky, and possibly @rachelnicole?)
+
+
+### Central communication medium for chatting in Node.js [#11](https://github.com/nodejs/community-committee/issues/11)
+IRC vs. Slack!
+How do we make this decision?
+
+CTC, existing moderators, and community-at-large should each get a separate input mechanism. 
+
+Why not have both exist? Are we creating this for the collaborators? For the community? What about the moderators? 
+
+Is this really a support issue? Are we properly supporting our community? nodejs/help is a great way for people to answer questions, but this isn’t realtime feedback. Can we hook /help into a chat but then make sure there’s protocol for making /help the source of truth for that answer for others to reference later?
+
+Can we let the community build their spaces? 
+
+To do (@hackygolucky):
+- Make an official, CoC compliant list of places we can send folks needing help!
+- Document what problems we are trying to solve
+- Update the cross-posted issue that we propose not to choose one over the other
+- Moderation training
+- Can we create a hook from /help into IRC and/or Slack for people to get smaller help in realtime
+
+## Q/A
+N/A
+
+## Next Meeting
+
+Node.js Foundation calendar: Coming Soon!

```

Comment:
@williamkapke @hackygolucky, I don't seem to have access to edit the calendar, can one of you make the next meeting entry?